test/run_test.py

```diff
@@ -108,6 +108,11 @@
     'test_fx_experimental',
     'test_functional_autograd_benchmark',
     'test_package',
+    'test_license',
+:qa
```

Comment:
I'm assuming this isn't meant to be there?

Reply:
whoops, fixed